documents/service.go

```diff
@@ -1,10 +1,25 @@
 package documents
```

Comment:
I agree with this change
but I have doubts whether we are making this service interface too specific for our current usecases, anyway we would never know until we have those uses.